src/serlio/utils/MELScriptBuilder.cpp

```diff
@@ -160,6 +160,14 @@ void MELScriptBuilder::getWorkspaceDir() {
 	commandStream << L"workspace -q -rd;\n";
 }
 
+void MELScriptBuilder::getStringFromURL(const std::wstring& url) {

```

Comment:
adding this method does not fit with the rest of the MELScriptBuilder, please move it elsewhere

Reply:
removed.